### ISSUE TITLE
Bot also memes urls

### ISSUE THREAD

**Issue by gorrillamcd** [lukehorak/chosenBot#3]
I've seen a couple examples of the bot spotting "men" in a URL and memeifying that, which makes for a really weird reply.

**Comment by gorrillamcd** [comment]
A related issue is that the bot will also reply to mentions of its own username, not_just_the_men. This is problematic when people understandably reply "good bot" because the chosenbot ends up having the same conversation over and over again with the voting bot, since it mentions the username.

**Comment by lukehorak** [comment]
Yeah, I'm still tinkering with the expression, but I Think I've got a fix
for it. Just working on a solution to stop it from re-commenting on posts
every time I redeploy it.
